auth/src/args.rs

```diff
@@ -15,7 +15,8 @@ pub struct Args {
 #[derive(Subcommand, Debug)]
 pub enum Commands {
     Start(StartArgs),
-    Init(InitArgs),
+    InitAdmin(InitArgs),
+    InitDeployer(InitArgs),
```

Comment:
nit: I thought about `InitMachineUser` that can have deploy rights.

Reply:
Hmm, interesting. I will prefer the keep the mapping to the tier type (`Deployer`) closer though